.github/workflows/gh-pages.yml

```
@@ -0,0 +1,96 @@
+name: Publish Web Bundle
+
+on:
+  # Our web bundle is published manually now,
+  # but this trigger is going to be changed to an event that guideline has been updated
+  push:
+    branches:
+    - publish-webbundle
+
+jobs:
+  build-publish:
+    runs-on: ubuntu-18.04
+    env:
+      TZ: Asia/Tokyo
+      BUNDLE_NAME: ameba-accessibility-guidelines.wbn
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        ref: ${{ github.ref }}
+
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    - name: Setup Hugo
+      uses: peaceiris/actions-hugo@v2.3.0
+      with:
+        hugo-version: 0.55.6
+
+    - name: Build CSS
+      run: |
+        npm install
+        npm run build
+    
+    - name: Build Hugo
+      run: hugo --minify
+
+    - name: Install go/bundle CLI
+      run: |
+        # Go has been installed at the setup hugo step,
+        # you can run setup-go action if you want to install specific version
+        go get -u github.com/WICG/webpackage/go/bundle/cmd/...
```

[nits] 不勉強なことに`go get`をそこまで知らないんですが、これでダウンロードできるんですね。末尾の`...`が気になる……

---

こりですかなぁ〜
https://stackoverflow.com/questions/28031603/what-do-three-dots-mean-in-go-command-line-invocations

よく考えたら、`gen-bundle` しか使わないから個別指定にしてよかったですね。

---

確かに。通信量節約できますね。

---

@sasaplus1 このworkflowで必要な、`gen-bundle` だけ取得するように変更しました。

```suggestion
        go get -u github.com/WICG/webpackage/go/bundle/cmd/gen-bundle
```